src/org/zalando/stups/magnificent/api.clj

```diff
@@ -16,19 +17,34 @@
 
 (defn- find-user
   [realm user request]
-  (let [user-api (get-in request [:configuration :user-api])
-        kio-api  (get-in request [:configuration :kio-api])
-        token    (get-in request [:tokeninfo "access_token"])]
-    (if (= realm "employees")
-      (user/get-human-user
-        user-api
-        user
-        token)
-      (if (= realm "services")
-        (user/get-robot-user
-          kio-api
-          user
-          token)))))
+  (let [user-api-url (get-in request [:configuration :user-api])
+        kio-api-url  (get-in request [:configuration :kio-api])
+        token        (get-in request [:tokeninfo "access_token"])]
+    (condp = realm
```

Comment:
Ah! I was looking for that, but only remembered `cond` and then went for `condp` :grin: